manifests/config.pp

```diff
@@ -16,8 +16,8 @@
 
   concat{ $candlepin::candlepin_conf_file:
     mode  => '0600',
-    owner => 'tomcat',
-    group => 'tomcat',
+    owner => $candlepin::user,
+    group => $candlepin::group,
```

Comment:
Unrelated fix, but thoughts on making this root:$group with mode 640? It would remove the write bit from the service which I think is a good thing.